paasta_tools/cli/cmds/logs.py

```diff
@@ -637,6 +637,15 @@ def __init__(self, cluster_map: Mapping[str, Any]) -> None:
             )
         self.cluster_map = cluster_map
 
+    def get_scribereader_selector(self, scribe_env: str) -> str:
+        # this is kinda silly, but until the scribereader cli becomes more ergonomic
+        # we'll need to do a little bit of string munging to let humans use scribereader
+        # in the same way we are (tl;dr: scribereader has sorta confusing behavior between
+        # what can be use for --ecosystem, --region, and --superregion and the fastest/least
+        # hacky thing to figure out which we wanna use is that any env with a - in it is a region
+        # and any without one is an ecosystem)
+        return "-e" if "-" in scribe_env else "-r"
```

Comment:
There might've been some shenanigans using `scribe_env_to_locations` since this checks if this is a region/superregion/ecosystem for us by looking it up in the `scribereader.PROD_REGIONS`/`scribereader.PROD_SUPERREGIONS` lists?

Reply:
oh, that would have been another good way to do this!

!choose use the simple yet effective way, do things in a more verbose manner so that i don't need to write a comment